packages/design-tokens/src/tokens/functional/components/faq/colors.json

```diff
@@ -1,10 +1,28 @@
 {
   "brand": {
     "FAQ": {
-      "heading": {
-        "color": {
+      "color": {
+        "heading": {
           "value": "var(--brand-color-text-default)",
           "dark": "var(--brand-color-text-default)"
+        },
+        "tabBorder": {
+          "selected": {
+            "value": "hsl(var(--base-color-scale-blue-5-hsl) / 20%)",
+            "dark": "hsl(var(--base-color-scale-blue-4-hsl) / 20%)"
+          }
+        },
+        "tabLabel": {
+          "selected": {
+            "value": "var(--brand-Link-color-accent)",
+            "dark": "var(--brand-Link-color-accent)"
+          }
+        },
+        "tabBg": {
+          "selected": {
+            "value": "hsl(var(--base-color-scale-blue-5-hsl) / 08%)",
+            "dark": "hsl(var(--base-color-scale-blue-4-hsl) / 08%)"
```

Comment:
@jesussandreas @josepmartins -- I'm confused. In our Enterprise comp, we're not using a blue highlight at all. Can we make sure that we're following the patterns outlined in our Landing Pages so there's no discrepancies? 
![Screenshot 2023-09-08 at 13 37 14](https://github.com/primer/brand/assets/63429045/9349cad7-4902-46a5-b23c-807b52ae43d1)

Reply:
i think it's because initially the blue, it was thought to act as the accent color (ex: our default pink) that can change depending on the brand theme. but i can double check on this and perhaps we blue as our set color.

Reply:
@jesussandreas im mostly referring to the left hand groups, not the accordion icons. I worry about using brand themes to impact active states. The blue makes sense here, but I'd love to have a gray default set here in case we want to reduce the volume.

Reply:
I added that to the tracking issue https://github.com/github/primer/issues/2510, as this PR was merged last week so we need to update Figma and React libraries.